eng/Versions.props

```diff
@@ -1,12 +1,26 @@
-﻿<Project>
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MajorVersion>3</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseLabel>preview</PreReleaseLabel>
+    <PreReleaseVersionLabel>$(PreReleaseLabel)</PreReleaseVersionLabel>
```

Comment:
Is this an Arcade thing? Can we just rename PreReleaseLabel to PreReleaseVersionLabel? I think Arcade implements a lot of the funky MSBuild version match in this file and we may not need to repeat it.